OrbitCore/CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ target_sources(
           OrbitThread.cpp
           Params.cpp
           Path.cpp
+          Pdb.cpp
```

Comment:
I take it the end goal is to remove this completely.

Reply:
Yes